src/components/Calendar/Calendar.tsx

```diff
@@ -96,7 +107,7 @@ const Calendar: React.FC<CalendarProps> = ({
                         center: "title",
                         right: "dayGridMonth,timeGridWeek,timeGridDay",
                     }}
-                    events={initialEvents}
+                    events={endDateInclusiveEvents(initialEvents)}
```

Comment:
name could be `makeAllDayEventsInclusive` or similar, `endDateInclusiveEvents` to me doesn't really make sense, especially if it runs on the whole event object

Reply:
refactored as suggested